NEWS.md

```diff
@@ -6,13 +6,8 @@ Version 0.19 (unreleased)
 
 **JuMP 0.19 contains significant breaking changes.**
 
-We will tag alpha and beta versions of 0.19 as appropriate to make it easier to
-test JuMP 0.19 before the final release. These are preview releases for early
-adopters. Additional breaking changes may occur between development tags.
-The documentation has known gaps. When using a development tag, be prepared to
-peek into the JuMP source code and tests for examples of how things work.
-
-Latest development tag: `v0.19-beta2` (`] add JuMP#v0.19-beta2`).
+We invite earlier adopters to try out the `v0.19-beta2` release
```

Comment:
Why keep that ? It will be in the README of JuMP v0.19.0 installations

Reply:
However, since the README is hidden in `.julia/packages/???/JuMP/README.md` it's might be ok to keep it and remove it after the tag

Reply:
This sentence should be removed before the final tagging.